Makefile

```diff
@@ -618,6 +618,9 @@ browse-site: build/site/
 deploy-dev-site: build/site/
 	cd $(BUILD_DIR)/site && gcloud $(OM_SITE_GCP_PROJECT_FLAG) app deploy .app.yaml --promote --version=$(VERSION_SUFFIX) --quiet
 
+deploy-redirect-site:
```

Comment:
Should this be a phony?  I think you added phonys in a different uncommitted change?